wordpress/wp-content/plugins/sil-dictionary-webonary/include/browseview_func.php

```diff
@@ -851,13 +860,20 @@ function vernacularalphabet_func( $atts )
 
 		//$arrPosts = query_posts("s=a&letter=" . $chosenLetter . "&noletters=" . $noLetters . "&langcode=" . $languagecode . "&posts_per_page=" . $posts_per_page . "&paged=" . $_GET['pagenr'] . "&DisplaySubentriesAsMainEntries=" . $displaySubentriesAsMinorEntries);
 		$pagenr = filter_input(INPUT_GET, 'pagenr', FILTER_VALIDATE_INT, array('options' => array('default' => 1)));
-		$arrPosts = getVernacularEntries($chosenLetter, $languagecode, $pagenr);
 
+		if( get_option('useCloudBackend') )
+		{
+			$dictionary = Webonary_Cloud::getBlogDictionaryCode();
+			$arrPosts = Webonary_Cloud::getEntriesAsPosts(Webonary_Cloud::$doBrowseByLetter, $dictionary, $chosenLetter);
```

Comment:
This is a potential memory problem, depending on how many posts are returned. However, it's no worse than the existing code, so no change is required.

Reply:
This will be somewhat mitigated once we add paging, so only an X number of posts will be returned per call.